src/controllers/search-product-name.controller.js

```diff
@@ -0,0 +1,22 @@
+const searchProductService = require("../services/search-product-name.service");
+
+const searchProducT = async(req,res)=> {
```

Comment:
```suggestion
const searchProducByName = async(req,res)=> {
```
이전 PR에서 남겨드렸던 코멘트와 같은 내용입니다. 모든 naming은 그 내부 동작 혹은 담을 데이터에 어떠한 것인지 이름만으로 알 수 있도록 지어주세요.